javascripts/AppConfig.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+app.run(function(FIREBASE_CONFIG) {
+	firebase.initializeApp(FIREBASE_CONFIG);
+});
+
+
+app.config(function($routeProvider){
+  $routeProvider
+    .when("/login", {
+      templateUrl: 'partials/login.html',
+      controller: 'loginCtrl'
+    })
+    .when("/view", {
```

Comment:
This route should be for `/contacts/view` not `view` (same with the 2 below it).

Reply:
@cmurnick - Make sure you are changing your ROUTE (or the `when(XXX)`). The route is what has the direct correlation with the navbar links. The route name and the partial you are using don't have a direct name relationship, it just looks at what you tell it to load in the templateUrl and does that, even if the templateUrl and the route have very different names. 

Holla if you have questions about this and push up when you have changes to your routes.